src/webview_provider/altimateWebviewProvider.ts

```
@@ -223,6 +243,15 @@ export class AltimateWebviewProvider implements WebviewViewProvider {
 
     try {
       switch (command) {
+        case "getTeammatesStatus":
+          const isEnabled = workspace
+            .getConfiguration("dbt")
+            .get<boolean>("enableTeammates", false);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)